index.js

```diff
@@ -1,11 +1,19 @@
 const beautify = require('pretty')
 
+const isHtmlString = received => typeof received === 'string' && received[0] === '<'
+const isVueWrapper = received => (
+  typeof received === 'object' &&
+  typeof received.isVueInstance === 'function' &&
```

Comment:
@eddyerburgh I made the change as you suggested. I am wondering if you think it's necessary to actually call isVueInstance when the existence of the method is already Vue-specific. That is, of course, assuming this method will always return true. I put both the existence check and the call in for now, but will defer to your judgement.

(Repeating this comment due to the previous one being shown as outdated due to a squashed commit).

Reply:
Great, thanks. I've pushed that change.